docker-compose.yml

```diff
@@ -40,30 +44,35 @@ services:
     image: copilot-api
     ports:
       - "8000:8000"
+    env_file:
+      - .env
+    environment:
+      - CORS_ALLOWED_ORIGINS=${CORS_ALLOWED_ORIGINS}
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+      - POSTGRES_HOST=${POSTGRES_HOST}
     volumes:
-      - ./api/app:/app    
-    networks: 
+      - ./api/app:/app
+    networks:
       - copilot-network
     depends_on:
       - db
       - chatbot
 
   # frontend
-  streamlit:
-    build:
-      context: ./frontend
-      dockerfile: Dockerfile
-    image: copilot-frontend
+  chatgpt-minimal:
+    build: ../chatgpt-minimal
```

Comment:
We are doing it like this at the moment, but we should discuss this point of a separate repository for mvp02. The important thing is that anyone can start a github workspace without forking other repos first. So I think we need a docker that imports our fork by URL path.

Reply:
Yes, I had some issues with integrating a git clone of our chatgpt-minimal fork in the frontend Dockerfile of the eak-copilot repo. I will create an other issue to deploy the frontend only from the eak-copilot project, and remove the necessity of manually cloning our fork of the chatgpt-minimal repo.

Reply:
@K-Schubert ok. we do this in sprint-02. not important yet. important we have a issue to ckeck this later